packages/pages/src/common/src/template/internal/types.ts

```diff
@@ -57,13 +58,20 @@ export interface TemplateConfigInternal {
   /** The stream that this template uses. If a stream is defined the streamId is not required. */
   streamId?: string;
   /** The stream configuration used by the template */
-  stream?: Stream;
+  stream?: Stream<LocalizationInternal>;
```

Comment:
You probably want a Stream and StreamInternal. The external types shouldn't rely on the Internal ones.